packages/perseus/src/util/parse-perseus-json/perseus-parsers/interactive-graph-widget.ts

```diff
@@ -276,18 +277,23 @@ export const parseInteractiveGraphWidget: Parser<InteractiveGraphWidget> =
         constant("interactive-graph"),
         object({
             step: pairOfNumbers,
-            gridStep: pairOfNumbers,
-            snapStep: pairOfNumbers,
+            gridStep: optional(pairOfNumbers),
+            snapStep: optional(pairOfNumbers),
```

Comment:
I'd prefer it if we could default `gridStep` and `snapStep` here, but the defaulting logic used in the `InteractiveGraph` component is fairly complicated and depends on the graph bounds. I guess there should be a TODO comment here.